.github/workflows/publish-release.yml

```diff
@@ -14,16 +14,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Install JDK 8
+      - name: Install JDK
         uses: actions/setup-java@v1
         with:
           java-version: 11
 
       - name: Install Android SDK
         run: ./.github/android-sdk.sh
 
-      - name: Build release and docs
-        run: ./gradlew assemble dokkaHtml javaDocDebugJar javaDocReleaseJar
+      # publish to local repository so that everything can be built in parallel
+      - name: Publish locally
+        run: ./gradlew publishToMavenLocal
+        env:
+          ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.MAVEN_SIGNING_PRIVATE_KEY }}
+          ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.MAVEN_SIGNING_PASSWORD }}
```

Comment:
Using `publishToMavenLocal` is more effective and really covers everything compared to just running some of the build tasks